docs/exchange-operators/rosetta/run-with-docker.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Running with Docker
+description: Run Mina’s implementation of Rosetta API with Docker.
+keywords:
+  - historical data
+  - blockchain integration
+  - explorer
+  - wallet
+  - archive node
+  - graphQL
+  - rosetta
+  - docker
+---
+
+
+# Running with Docker
+
+:::info
+
+There is a known issue that you can't use the docker image on the Apple Silicon chip.
+If you want to run Mina Rosetta on the Apple Silicon, please follow steps described in [Building and running Rosetta from source code](build-from-sources) page
+
+:::
+
+1. [Install Docker](https://www.docker.com/get-started) and check that your Docker configuration has at least 12 GB RAM (the recommended amount is 16 GB).
+
+2. Check the latest release for mainnet on the official [Mina github releases page](https://github.com/MinaProtocol/mina/releases). 
```

Comment:
```suggestion
2. Check the latest release for Mainnet on the official [Mina GitHub releases](https://github.com/MinaProtocol/mina/releases) page. 
```

Reply:
hi @qwadratic  did you know we have a word list? (we always misspell GitHub) https://github.com/o1-labs/docs2/wiki/Word-list